pkg/operator/discover/discover.go

```diff
@@ -298,6 +299,16 @@ func ListDevicesInUse(context *clusterd.Context, namespace, nodeName string) ([]
 	return devices, nil
 }
 
+func matchFullPath(dls, fp string) bool {
```

Comment:
would you mind either renaming the variables to be descriptive or add comments about dls and fp?

Reply:
done